docs/src/main/asciidoc/health-indicator.adoc

```diff
@@ -0,0 +1,9 @@
+## Health Indicator
+
+The client library comes with a Health Indicator that check whether the connection to the Azure App Configuration store(s) is healthy or not. If enabled, for each store it gives a status of:
+
+ * UP - The last connection was successful
```

Comment:
could you also state what actions can be taken by a customer if they see each of these? for example, for the down connection, you should follow the steps found at (link) or send another request etc.

Reply:
I am not sure which steps we would suggest for down as the client library would be retried. Or it may be a service issue. Do we have other docs for this?

Reply:
I don't know any specific docs for this, but we could at least append the sentence and say something like ".., this could be due to a number of issues, such as the client library being retired or there being a service issue"